UPGRADE_NOTES.md

```diff
@@ -1,3 +1,40 @@
+### Upgrading from 0.14 to 0.15
+
+- Upgrade to crystal 0.29.0
+- Upgrade Lucky CLI (macOS)
+
+```
+brew update
+brew upgrade crystal-lang # Make sure you're up-to-date. Requires 0.29.0
+brew upgrade lucky
+```
+
+- Upgrade Lucky CLI (Linux)
+
+- Update `.crystal-version` to `0.29.0`
+
+> Remove the existing Lucky binary and follow the Linux
+> instructions in this section
+> https://luckyframework.org/guides/getting-started/installing#install-lucky
```

Comment:
We can link to https://luckyframework.org/guides/getting-started/installing#on-linux to direct to the specific Linux section